tempesta_fw/http.c

```diff
@@ -831,8 +879,13 @@ tfw_http_req_fwd_send(TfwSrvConn *srv_conn, TfwServer *srv,
 	if (tfw_connection_send((TfwConn *)srv_conn, (TfwMsg *)req)) {
 		TFW_DBG2("%s: Forwarding error: conn=[%p] req=[%p]\n",
 			 __func__, srv_conn, req);
-		tfw_http_req_error(srv_conn, req, equeue, 500,
-				   "request dropped: forwarding error");
+		if (req->flags & TFW_HTTP_HMONITOR) {
+			tfw_http_req_delist(srv_conn, req);
+			tfw_http_msg_free((TfwHttpMsg *)req);
```

Comment:
So, if we're unable to send a health monitoring request, then it's simply dropped without any trace or action upon that? Is that the intended behaviour?

Reply:
Warning added.